build.sbt

```diff
@@ -74,7 +74,6 @@ lazy val standardSettings = Seq(
     "UTF-8",
     "-Xlint",
     "-deprecation",
-    "-Xfatal-warnings",
```

Comment:
The `fatal-warnings` flag is too important to remove it.
If you cannot find a way to let the tests involving deprecation pass, I would rather delete them that removing the flag.